bin/run-browserstack-tests.mjs

```diff
@@ -36,13 +38,13 @@ function run(command, args = []) {
       ]);
 
       console.log('success');
-      // eslint-disable-next-line n/no-process-exit
-      process.exit(0);
     } finally {
       if (process.env['GITHUB_RUN_ID']) {
         await run('ember', ['browserstack:results']);
       }
       await run('ember', ['browserstack:disconnect']);
+      // eslint-disable-next-line n/no-process-exit
+      process.exit(0);
```

Comment:
The previous code ran process.exit() in the `try`, which would mean that the `finally` would never
run. I'm not sure why that worked (perhaps because we never relied on browserstack to run
properly?), but I moved the `exit` to the end of the `finally` rather than completely delete it,
since it presumably exists for a reason?